debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -456,6 +464,20 @@ public boolean isSkipLowActivityLsnsEnabled() {
         return skipLowActivityLsnsEnabled;
     }
 
+    public Set<Envelope.Operation> getSkippedOps() {
```

Comment:
Isn't this method already inherited from `io.debezium.config.CommonConnectorConfig#getSkippedOps` ?

Reply:
Yes! I don really remember why I did this I had some problems with that. I do my best to do it right